src/translations/en.json

```diff
@@ -827,32 +827,8 @@
             "reloading": {
               "heading": "YAML configuration reloading",
               "introduction": "Some parts of Home Assistant can reload without requiring a restart. Hitting reload will unload their current YAML configuration and load the new one.",
-              "core": "Reload location & customizations",
-              "group": "Reload groups",
-              "automation": "Reload automations",
-              "script": "Reload scripts",
-              "scene": "Reload scenes",
-              "person": "Reload persons",
-              "zone": "Reload zones",
-              "input_boolean": "Reload input booleans",
-              "input_text": "Reload input texts",
-              "input_number": "Reload input numbers",
-              "input_datetime": "Reload input date times",
-              "input_select": "Reload input selects",
-              "template": "Reload template entities",
-              "universal": "Reload universal media player entities",
-              "rest": "Reload rest entities",
-              "command_line": "Reload command line entities",
-              "filter": "Reload filter entities",
-              "statistics": "Reload statistics entities",
-              "generic": "Reload generic IP camera entities",
-              "generic_thermostat": "Reload generic thermostat entities",
-              "homekit": "Reload HomeKit",
-              "min_max": "Reload min/max entities",
-              "history_stats": "Reload history stats entities",
-              "trend": "Reload trend entities",
-              "ping": "Reload ping binary sensor entities",
-              "filesize": "Reload file size entities"
```

Comment:
Should we keep the translations that do not result in an identical string?

Reply:
Yeah we can, should we leave all in? They all are slightly different I think, mainly because of plurals

Reply:
yeah, why not.
We can always change it / remove some in the future.

Reply:
@bramkragten if possible please add rpi_gpio to the translations (https://github.com/home-assistant/frontend/pull/6764/files#diff-571ce32a17d6aa22fdd0942224a8ff44R856)

`"rpi_gpio": "Reload Raspberry Pi GPIO entities"`

Reply:
What does it say now?

Reply:
If it needs to change, please create a PR

Reply:
currently, there is no rpi_gpio entry.
I'll check how this looks and if needed create a new PR.